src/components/newTutorialForm/NewTutorialForm.tsx

```diff
@@ -0,0 +1,87 @@
+import './NewTutorialForm.css'
+
+import axios from 'axios'
+import React, { useState } from 'react'
+
+import InputField from '@/components/inputField/InputField'
+
+const NewTutorialForm: React.FC = () => {
+  // TODO use one state for all input values somehow xD
```

Comment:
Using class component maybe :)

Reply:
The problem is typescript, as soon as I understand how to made an interface I'll create some custom hook for input control.